tests/samplers_tests/tpe_tests/test_truncnorm.py

```diff
@@ -0,0 +1,37 @@
+import sys
+
+import numpy as np
+import pytest
+from scipy.stats import truncnorm as truncnorm_scipy
+
+import optuna.samplers._tpe._truncnorm as truncnorm_ours
+
+
+@pytest.mark.skipif(
+    sys.version_info < (3, 8, 0), reason="SciPy 1.9.2 is not supported in Python 3.7"
+)
+@pytest.mark.parametrize(
+    "a,b", [(-np.inf, np.inf), (-10, +10), (-1e-3, +1e-3), (10, 100), (-100, -10)]
```

Comment:
Could you add smaller `a` and `b` values? For example, `(-1, 1)`.

Reply:
added (-1, +1)